src/ending.c

```diff
@@ -65,59 +65,63 @@ void bad_ending_marisa(Ending *e) {
 
 	add_ending_entry(e, -1, "Marisa closed the book and let it slam onto the floor.", NULL);
 	add_ending_entry(e, -1, "Patchouli: “Mind your manners! That’s a very old tome!”", NULL);
-	add_ending_entry(e, -1, "Marisa: “It’s so irritatin’! I thought I solved everything but now that tower’s just sittin’ there, ’n the whole Mountain’s been quarantined…”", NULL);
-
+	add_ending_entry(e, -1, "Marisa: “It’s so irritatin’! I thought I solved everything but now that tower’s just sittin’ there, ‘n the whole Mountain’s been quarantined…!”", NULL);
 	add_ending_entry(e, -1, "Marisa: “It just ain’t satisfyin’, y’know?”", NULL);
-	add_ending_entry(e, -1, "Flandre: “I could destroy it! Make it go BOOM!”", NULL);
-	add_ending_entry(e, -1, "Marisa: “Gah! Now what did I say about sneakin’ up on people, kiddo?”", NULL);
-	add_ending_entry(e, -1, "Flandre: “Sorry!”", NULL);
+
+	add_ending_entry(e, -1, "Flandre: “I could destroy it, you know.”", NULL);
+	add_ending_entry(e, -1, "Marisa: “Gah! Now what did I say about sneakin’ up on your friends, Flan?”", NULL);
+	add_ending_entry(e, -1, "Flandre: “Sorry, sorry!”", NULL);
 	add_ending_entry(e, -1, "Patchouli: “Thank you for the offer, Flandre, but no.”", NULL);
 	add_ending_entry(e, -1, "Flandre: “Awww…”", NULL);
 
 	add_ending_entry(e, -1, "After finishing tea with the pair of them, Marisa wandered the stacks, looking for inspiration.", NULL);
-	add_ending_entry(e, -1, "Somehow, she tripped on her own feet and landed face-first against a particularly hard and sharp-edged book.", NULL);
-	add_ending_entry(e, -1, "Upon picking it up, she noticed it was actually an ’Intelligent Device’ of some kind, like those portable computers from the Outside World.", NULL);
-	add_ending_entry(e, -1, "This one was far more advanced than anything she’d ever seen.", NULL);
+	add_ending_entry(e, -1, "Somehow, she tripped on her own feet, almost as if something had grabbed her foot, and landed face-first against a particularly hard and sharp-edged book.", NULL);
```

Comment:
Not sure it's a necessary addition, but I'm not really against it either.

I'd suggest maybe
```suggestion
	add_ending_entry(e, -1, "Somehow, she tripped on her own feet, almost as if something had grabbed her, and landed face-first against a particularly hard and sharp-edged book.", NULL);
```
to not repeat feet/foot in the same sentence.

Reply:
Revised the entry to be more ~mysterious~.